### PR TITLE
scylla_cloud_io_setup:fix wrong function usage for i4i instances

### PR DESCRIPTION
**Pull request by yaronkaikov**
When i cherry-pick 4d7c34bdb8216b1f0664d53df8b02a3e8b6eb92b , i forgot to update the function from `instancetype` to `instance()`

Closes: https://github.com/scylladb/scylla-machine-image/issues/365

### CODE REVIEW

**Comment by benipeled** [comment]
@yaronkaikov LGTM
why are these differences between master and branch-5.0
`lib/scylla_cloud_io_setup.py` --> `common/scylla_cloud_io_setup.py`

**Comment by yaronkaikov** [comment]
> @yaronkaikov LGTM why are these differences between master and branch-5.0 `lib/scylla_cloud_io_setup.py` --> `common/scylla_cloud_io_setup.py`

We did some changes in master and once we branch we don't merge/backport changes. in this case we had to add `i4i` support, but we didn't backport all the commits before.

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**
